Documentation/CHANGES.md

```diff
@@ -1,7 +1,51 @@
 ## Summary
 
+This version brings a few bug fixes and updates to our v2.0.0 release.
+
 ## New features
 
+- Improve logging system and add `HG_LOG_SUBSYS` environment variable that can be used
+in combination of `HG_LOG_LEVEL` to select log sub-systems.
+  - Add `min_debug` log level to keep debug traces. Traces get printed when an error occurs.
+  - [HG] Add `HG_Set_log_subsys()`.
+- [NA] Add support for message size hints though max_unexpected_size and max_expected_size hints. Supported with OFI/BMI and MPI plugins.
```

Comment:
"with OFI/BMI and MPI" -> "with OFI, BMI, and MPI"?  (OFI/BMI is not a valid combiation)